### ISSUE TITLE
Accessibility issue in wizard - The section in wizard content does not have a role attribute 

### ISSUE THREAD

**Issue by MonishaK28** [SAP/fundamental-ngx#12674]
![image](https://github.com/user-attachments/assets/e7b25e4a-3075-448f-a15f-257e421f7784)

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.) 
Latest

#### Is there anything else we should know?

**_IMPORTANT: Please refrain from providing links or screenshots of SAP's internal information, as this project is open-source, and its contents are accessible to anyone._**
